src/middlewared/middlewared/utils/osc/freebsd/user_context.py

```diff
@@ -86,6 +86,13 @@ def run_command_with_user_context(commandline, user, callback):
             callback(get)
         except queue.Empty:
             pass
+        except Exception as e:
+            logger.error(
+                'Error encountered: %r', e, exc_info=True
```

Comment:
We don't need `'%r' % e`, when we use `exc_info=True`. Also let's make error message something like `Unhandled exception`